python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -1026,6 +1026,7 @@ class BuiltinDataset(OnDiskDataset):
         "ogbn-papers100M-seeds",
         "igb-hom-medium",
         "igb-hom-medium-seeds",
+        "igb-hom-large-seeds",
```

Comment:
To remain consistent with other datasets:
```suggestion
        "igb-hom-large",
        "igb-hom-large-seeds",
```